plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2986,6 +2986,7 @@ elementgui.potion.amplifier=Amplifier:
 elementgui.potion.ambient=Ambient?
 elementgui.potion.show_particles=Show particles?
 elementgui.potion.error_potion_needs_display_name=Potion needs a display name
+elementgui.procedure.skip_dependency_null_check=<html>Skip null check for dependencies:<br><small>This option should be false in most cases
```

Comment:
"This option should be unchecked in most cases"

better than false because most users should leave this unchecked won't know what "false" is